lattice-control/lattice-control-interface.smithy

```diff
@@ -348,14 +397,16 @@ structure ProviderDescription {
 /// A command sent to a specific host instructing it to start the actor
 /// indicated by the reference.
 structure StartActorCommand {
-    /// Reference for the actor. Can be any of the acceptable forms of unique identification
+    /// The ID of the lattice on which this request will be performed
     @required    
-    @serialization(name: "actor_ref")
+    latticeId: String,    
+
+    /// Reference for the actor. Can be any of the acceptable forms of unique identification
+    @required        
     actorRef: String,
```

Comment:
While we're here, this comment should indicate that it's where you can start the actor from, not unique identifier

Reply:
Either image ref or bindle URL